source/manual/zendesk.html.md

```diff
@@ -0,0 +1,88 @@
+---
+
+2nd line Zendesk tickets are technical errors reported by our users - including government publishers -
+and general queries from users of data.gov.uk, for whom 2nd line provides 1st line support.
+
+## Get started
+
+[Create an account](zendesk-create-account), then ask a fellow 2nd liner to assign a new ticket to
+"2nd/3rd Line—Zendesk Administration" asking to give you access to "2nd Line - GOV.UK Alerts and Issues".
+
+When you're logged in, you should be looking at the ['2nd Line--GOV.UK Alerts and Issues' queue][zendesk-queue].
+
+## Priorities
+
+Work on 'High' priority tickets first - these will have been triaged by 2nd line already, so should be
+relevant and actionable - and start with the oldest by date. If external assistance from outside 2nd
+line is required, assign them on the ticket and move on to the next ticket (keeping the ticket in the
+2nd line queue).
+
+Ignore 'Low' priority tickets - these have been triaged by 2nd line for GOV.UK product managers to review.
+
+New tickets are 'Normal' priority by default and require triaging (i.e. a developer reading and categorising
+it).
+
+## Triaging a ticket
+
+If you have any doubts as to the legitimacy or urgency of a ticket, it should be deferred to the delivery manager.
+
+Once you've established that a ticket is legitimate, you should follow [the process in this diagram](zendesk-triage-diagram).
+There is a printout on the 2nd line desk. In general:
+
+- Questions from users of data.gov.uk can be answered straight away (we're their first line of support)
+- General queries are usually handled by 1st line and content support
+- Product requests should be changed to 'Low' priority and assigned to the product owner of the most relevant
+  product (you can find ownership information under [Applications on GOV.UK](https://docs.publishing.service.gov.uk/apps.html))
+- Technical problems to investigate should be reprioritised as "High" and dealt with by 2nd line
+
+In all cases, 2nd line will respond to the ticket, even if it is simply a note to the user saying that
+you've passed their request to the relevent team.
+
+### Service Level Agreements
+
+GOV.UK have committed to a minimum service level agreement for 2nd line support.
+
+* 80% of tickets get a first reply to the ticket within 2 working days
+* 80% of general enquiries (public tickets) resolved within 5 working days
+* 70% of department tickets are closed within 5 working days
+
+If the volume of Zendesk tickets is overwhelming, talk to the delivery manager or a senior tech member
+for assistance.
+
+## Picking up a ticket
+
+Click the "take it" link under "Assignee", followed by "Submit as Open", to formally undertake a ticket.
+If you know you cannot solve a ticket immediately, fill in a "Public reply" to the requester to let them
+know you’re looking into it.
+
+Before attempting to solve the problem yourself, read [Common 2nd line support requests for data.gov.uk](/manual/data-gov-uk-2nd-line.html),
```

Comment:
This manual page only applies to data.gov.uk tickets, however the text here reads that it could apply to all requests.